TOC.md

```diff
@@ -116,16 +116,32 @@
     + [软件版本](/tune-software-version.md) @张文博
     + 配置
       + [TiKV 调优](/tune-tikv-performance.md) @刘玮
-  + SQL 性能调优
-    + [调优基本概念](/sql-optimization-concepts.md) @崔一丁
-    + 优化器概览
-      + [执行计划](/query-execution-plan.md) @崔一丁
-      + [分布式执行计划](/distributed-exectuion-plan.md) @崔一丁
-      + 管理执行计划
-        + [执行计划绑定](/execution-plan-binding.md) @崔一丁
-        + [Optimizer Hints](/optimizer-hints.md) @崔一丁
-        + [使用 Index Merge 方式访问表](/index-merge.md) @崔一丁
-      + [Join Reorder 算法简介](/join-reorder.md) @崔一丁
+    + [SQL 性能调优] @崔一丁
+      + [SQL 性能调优概览](/sql-tuning-overview.md)
+      + [理解 TiDB 执行计划](/query-execution-plan.md)
+      + [SQL 优化]
+        + [SQL 优化流程简介](/sql-optimization-concepts.md)
+        + [逻辑优化]
+          + [逻辑优化概览](/sql-logical-optimization.md)
+          + [子查询相关的优化](/subquery-optimization.md)
+          + [列裁剪](/column-pruning.md)
+          + [关联子查询去关联](/correlated-subquery-optimization.md)
+          + [Max/Min 消除](/max-min-eliminate.md)
+          + [谓词下推](/predicate-push-down.md)
+          + [分区裁剪](/partition-pruning.md)
+          + [TopN 和 Limit 下推](/topn-limit-push-down.md)
+          + [Join Reorder](/join-reorder.md)
```

Comment:
我们目前有下面这些 optimize rule:
```go
var optRuleList = []logicalOptRule{
	&gcSubstituter{},
	&columnPruner{},
	&buildKeySolver{},
	&decorrelateSolver{},
	&aggregationEliminator{},
	&projectionEliminator{},
	&maxMinEliminator{},
	&ppdSolver{},
	&outerJoinEliminator{},
	&partitionProcessor{},
	&aggregationPushDownSolver{},
	&pushDownTopNOptimizer{},
	&joinReOrderSolver{},
	&columnPruner{}, // column pruning again at last, note it will mess up the results of buildKeySolver
}
```
这里像 外连接消除、投影消除等 都是没有提到的，我们要考虑在文档加入这些吗？

Reply:
人手不一定够？不如我们先写完现有这一批，后面再加一批

Reply:
🉑️